Assignments/Program.cs

```diff
@@ -1 +1,46 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T13 - Strong Password
+// ---------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T15 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Check for Isogram Word:-" + "\x1B[0m");
+      Console.WriteLine ("An isogram is a word that has no duplicate letters.");
+      string output;
+      List<string> testCases = new () { "apple", "banana", "cat", "doctrine", "elephant", "fruits", "pebble", "rubiks", "yellow", "zebra" };
+      Console.WriteLine ();
+      Console.WriteLine ("\x1B[4m" + "Test Cases:" + "\x1B[0m");
+      for (int i = 0; i < testCases.Count; i++) {
+         Console.Write ($"{i + 1}. {testCases[i]} - ");
+         (output, Console.ForegroundColor) = IsIsogram (testCases[i]) ? ("Isogram", Green) : ("Not an Isogram", Red);

```

Comment:
You are accessing testCases[i] in this line and previous line.
Instead could store in a variable and reuse it.